core/jreleaser-engine/src/main/java/org/jreleaser/packagers/BrewPackagerProcessor.java

```diff
@@ -138,6 +138,9 @@
     private static final String FORMULA_MULTI_RB = "formula-multi.rb";
     private static final String RB = ".rb";
     private static final String SKIP_JAVA = "skipJava";
+    private static final String USE_VERSIONED_JAVA = "useVersionedJava";
+
+    private static final List<String> BREW_JDK_ALIASES = List.of("openjdk", "java");
```

Comment:
Can't use this API call because codebase mist remain Java8 compatible. Use `Collections.singleton()` instead.